docs/mining/how-to-stake.md

```diff
@@ -1,37 +1,40 @@
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/Pool.svg" /> PoS using a Stakepool
+## <img class="dcr-icon" src="/img/dcr-icons/Pool.svg" /> PoS using a Voting Service Provider (VSP)
+
+A list of Voting Service Providers (VSPs) and statistics is maintained on the
+[:fa-external-link-square: Decred.org website](https://decred.org/stakepools/).
 
-You can find the list of all official stakepools and statistics by visiting the
-[:fa-external-link-square: Decred website](https://decred.org/stakepools/).
+Using a VSP **does not give the pool access to your funds**. All you are doing is granting voting rights to the VSP.
 
-Using a stakepool DOES NOT give the pool access to your funds. All you are doing is granting voting rights to the pool.
+In order to support network decentralization, it is recommended that you join a smaller VSP with fewer live tickets. As VSPs control tickets delegated to them, they could in theory vote those tickets in a way which contradicts the expressed wishes of the ticket owners. This could be easily detected, and any VSP which attempted it would likely be abandoned by the stakeholder community. However, it is good practice to limit the power of individual VSPs, to limit the potential for damage from this kind of attack.
 
-In order to support network decentralization, it is recommended that you join a smaller pool with fewer live tickets.
+Unlike Proof-of-Work (PoW) mining pools, VSPs do not pool work or rewards. The number of tickets a VSP has does not affect how regularly one's tickets will be called and rewards received. 
 
-All stakepools run the same basic code, but they may differ in the amount of redundancy available.
-More redundancy equals less chance of missed votes, although all pools will have missed votes as many missed votes are caused by PoW miners.
-Sometimes they will find a solution to the blocks so quickly that votes haven't had time to propagate around the network.
+All VSPs run the same basic code, but they may differ in the amount of redundancy available.
+More redundancy equals less chance of missed votes, although all pools will have some missed votes as many missed votes are caused by PoW miners. This is because sometimes miners will find a solution to the next block so quickly that votes haven't had time to propagate around the network.
 
-There are currently two wallets which support stakepool voting:
+PoS voting using a VSP can be done using Decrediton or dcrwallet.
 
-- `Decrediton` - GUI wallet for Windows/macOS/Linux. The [Purchase Tickets tab](../getting-started/user-guides/using-decrediton.md#tickets) is used to buy tickets.
-- `dcrwallet` - CLI wallet for Windows/macOS/Linux. The [Buying Tickets with dcrwallet](../getting-started/user-guides/dcrwallet-tickets.md) guide explains how to purchase tickets via command line.
+- `Decrediton` - GUI wallet for Windows/macOS/Linux. The [Tickets menu, Purchase tab](/getting-started/user-guides/using-decrediton.md#tickets) is used to buy tickets.
```

Comment:
incorrect link - should be relative as it was before. same for the next line

Reply:
Added relative links, but I'm getting 404s...not sure why. Is this not the correct path?

../../getting-started/user-guides/using-decrediton.md#tickets
../../getting-started/user-guides/dcrwallet-tickets.md

Reply:
Fixed